spec/views/users/index.html.erb_spec.rb

```diff
@@ -1,5 +1,26 @@
 require 'rails_helper'
 
-RSpec.describe 'users/index.html.erb', type: :view do
-  pending "add some examples to (or delete) #{__FILE__}"
+RSpec.describe 'users/index.html.erb', type: :feature do
+  before :each do
+    @user = User.create(name: 'Koppai', photo: 'https://picsum.photos/id/23/200', bio: 'Micronaut')
+    @post = Post.create(author: @user, title: 'Rails set up', text: 'This is how')
+    visit root_path
+  end
+
+  scenario 'I can see the username' do
+    expect(page).to have_content(@user.name)
+  end
+
+  scenario 'I can see the profile picture' do
+    expect(page).to have_css("img[src*='https://picsum.photos/id/23/200']")
+  end
+
+  scenario 'I can see the number of posts' do
+    expect(page).to have_content("Number of posts: #{@user.post_counter}")
+  end
```

Comment:
- [ ] Well done for your progress so far in the test cases. However, kindly note that the 3 scenarios highlighted in your code above should check the following: 
1. You can see the username of ALL OTHER USERS.
2. You can see the profile picture for EACH USER
3. You can see the number of posts EACH USER has written..

![image](https://github.com/K0ppai/Blog_Web/assets/46839548/bf14f03c-f326-4624-891b-a76a05ba7d4b)